wp-admin/css/colors/_mixins.scss

```diff
@@ -4,31 +4,31 @@
  */
 @mixin button( $button-color, $text-color: white ) {
 	background: $button-color;
-	border-color: darken( $button-color, 10% );
+	border-color: darken( $button-color, 10% ) darken( $button-color, 15% ) darken( $button-color, 15% );
```

Comment:
Line should be indented with spaces, not tabs<br>Expected 0 spaces between parentheses instead of 1